File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string y=to_string(x);
        for(int i=0;i<y.size();i++){
            if(y[i]!=y[y.size()-1-i])
                return false;
        }
        return true;
    }
};

File: 0009-palindrome-number/0009-palindrome-number.cs
public class Solution {
    public bool IsPalindrome(int x) {
        String y=x.ToString();
        char[] c=y.ToCharArray();
        Array.Reverse(c);
        String r=new String(c);
        if(y==r)return true;
        return false;
    }
}